config/checkstyle_checks.xml

```diff
@@ -660,7 +660,8 @@
                        FileNotFoundException, TestException, Log, Sets, Multimap,
                        TokenStreamRecognitionException, RecognitionException,
                        TokenStreamException, IOException, Override, Deprecated, SafeVarargs,
-                       SuppressWarnings, FunctionalInterface"/>
+                       SuppressWarnings, FunctionalInterface, NoSuchMethodException,
+                       InvocationTargetException, IllegalAccessException"/>
```

Comment:
this is necessary because there is a violation in AutomaticBean class, there are multicatch for these exceptions